website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -564,7 +564,8 @@ describe('rendering', () => {
 
 <Info>
 
-In addition to the following options, the method accepts all options from `renderApp`.
+In addition to the following options, the method accepts all options from `renderApp`. 
+Please note that you can only pass one of `store`, `storeState`, or `sdkMocks`.
```

Comment:
```suggestion
Please note that it is not possible to pass either `storeState` or `sdkMocks` together with `store`.
```

Reply:
Yeah thanks, it was hard to figure out if it was only 1 of each or there was another combination that was possible.